customer-churn/README.md

```diff
@@ -174,7 +174,7 @@ Let's start by setting up our full AWS stack to run the pipeline using Seldon Co
 1. Install the Seldon Core integration, a set of ZenML extensions that integrate with Seldon Core.
 
 ```bash
-zenml integration install seldon
+zenml integration install seldon -y
```

Comment:
Btw, you don't always have to include the `-y` flag. All the `-y` flag does is it means you don't have to select yes or no as part of a confirmation dialogue.

Reply:
Sure, thanks